### ISSUE TITLE
Point is very hard to see

### ISSUE THREAD

**Issue by spegoraro** [arcticicestudio/nord-emacs#24]
Nice theme!

The current version on melpa, 20170325.2314, has an almost invisible point as seen in this screenshot:
![screenshot_20170403_192528](https://cloud.githubusercontent.com/assets/1013596/24607315/a903f784-18a3-11e7-8d07-116c779a1739.png)

**Comment by arcticicestudio** [comment]
Thanks for your report and nice to see you like it 👍 

Do you mean the cursor when you say "Point"? Else I can't see anything conspicuous on the screenshot.

**Comment by spegoraro** [comment]
Yep, that's what I mean. Setting its background to the foreground colour of the `default` face looks nice.

**Comment by arcticicestudio** [comment]
I've only set the `foreground` attribute instead of the `background` as documented.
> Basic face for the cursor color under X.
   Currently, only the ‘:background’ attribute is meaningful; all
   other attributes are ignored.
  The cursor foreground color is taken from the background color of the underlying text.

**Comment by spegoraro** [comment]
Yeah so if you change the cursor face line from:
```elisp
(cursor ((,class (:foreground ,nord4))))
```
to:
```elisp
(cursor ((,class (:background ,nord4))))
```
it will look fantastic!

**Comment by spegoraro** [comment]
Look great now! Thanks!

**Comment by arcticicestudio** [comment]
🚢 Shipped with MELPA package release version 🏷 [0.2.0](https://github.com/arcticicestudio/nord-emacs/releases/tag/v0.2.0).
